modules/workflow/src/main/java/org/eclipse/xpanse/modules/workflow/utils/WorkflowProcessUtils.java

```diff
@@ -110,12 +112,12 @@ public String getWorkFlowState(String processInstanceId) {
         if (instance == null) {
             if (variablesMap.containsKey(MigrateConstants.IS_RETRY_TASK)
```

Comment:
i think i missed this in the previous MRs. I would prefer getting the status from DB and I think we are even updating the status in DB. It is better to use that instead of doing any calculation based on the "retry" flag.